### PR TITLE
add support for voltage meter

### PR DESCRIPTION
**Pull request by githubcdr**
My meter has a voltage OBIS, I added this

### CODE REVIEW

File: src/dsmrp1.py [hunk 1]
```diff
@@ -25,6 +25,8 @@ def _unit(x):
 
 _tariff = lambda x: 'low' if x == '0002' else ('high' if x == '0001' else x)
 _id = lambda x: x
+_voltage = lambda x: x.split('*')[0]
```
Comment (bwesterb) [inline src/dsmrp1.py:28]: Why not use `_unit`?

**Review by bwesterb — CHANGES_REQUESTED** [review]
Thanks!  I have one question: why not use `_unit`?

File: src/dsmrp1.py (hunk 1 above)
Comment (githubcdr) [reply src/dsmrp1.py:28]: I don't know, I'm not a experienced python programmer, I wanted just the raw value not "(223.6*V)", feel free to improve!

Comment (bwesterb) [reply src/dsmrp1.py:28]: Ok, I'll write a patch tomorrow!

**Comment by bwesterb** [comment]
Added to commit d87afbef684d32487c494698873bd691e5f11b9e

Does it work?

**Pull request closed without merging**